src/dotnet/App.Server/appsettings.Development.json

```diff
@@ -72,6 +72,9 @@
         "ChatModel": "gpt-4o",
         "ApiKey": ""
       }
+    },
+    "Embeddings": {
+      "PredicationsUri" : "http://localhost:28080/predictions/Alibaba-NLP_gte-multilingual-base"
```

Comment:
Probably a typo in the "PredicationsUri" property name

Reply:
fixed